data/weapons/weapon_data.csv

```diff
@@ -1,59 +1,59 @@
-Micro Missile Launcher,diableavionics_microbuiltin,1,,500,1500,,100,,5,6,3,3,0.2,3,HIGH_EXPLOSIVE,50,,0.1,2.9,1,0,6,6,5,6,,300,180,12,25,,,"SYSTEM, DO_NOT_AIM, DO_NOT_CONSERVE, DO_NOT_AIM","utility10, missile1",,Diable Avionics,,Saturation,Medium,Poor,,,,,,,,
-Micro Missile Cluster,diableavionics_microcluster,3,,7500,1500,,100,,5,0,,30,0.5,6,HIGH_EXPLOSIVE,50,,0.2,4,6,0.1,6,6,5,6,,300,150,12,25,,,"SYSTEM, DO_NOT_AIM, DO_NOT_CONSERVE, DO_NOT_AIM",,,,,,,,,,,,,,,
-Micro Missile Cluster,diableavionics_microcluster_left,3,,7500,1500,,100,,5,0,,30,0.5,6,HIGH_EXPLOSIVE,50,,0.2,4,6,0.1,6,6,5,6,,300,150,12,25,,,"SYSTEM, DO_NOT_AIM, DO_NOT_CONSERVE, DO_NOT_AIM",,,,,,,,,,,,,,,
-Micro Missile Cluster,diableavionics_microcluster2,3,,7500,1500,,100,,5,0,,30,0.5,6,HIGH_EXPLOSIVE,50,,0.2,4,3,0.2,6,6,5,6,,300,5,15,25,,,"SYSTEM, DO_NOT_AIM, DO_NOT_CONSERVE, DO_NOT_AIM",,,,,,,,,,,,,,,
-Micro Missile Array,diableavionics_microarray,3,,17500,1500,,100,,5,0,,400,1.5,40,HIGH_EXPLOSIVE,50,,0.1,4,20,0.1,6,6,5,6,,300,150,12,25,,,"SYSTEM, DO_NOT_AIM, DO_NOT_CONSERVE, DO_NOT_AIM",,,,,,,,,,,,,,,
-Micro Missile Array,diableavionics_microarrayM,3,,17500,1500,,100,,5,0,,200,1.5,20,HIGH_EXPLOSIVE,50,,0.1,4,10,0.1,6,6,5,6,,300,150,12,25,,,"SYSTEM, DO_NOT_AIM, DO_NOT_CONSERVE, DO_NOT_AIM",,,,,,,,,,,,,,,
+Micro Missile Launcher,diableavionics_micromissile,1,,500,3000,,50,,5,6,3,25,1,4,HIGH_EXPLOSIVE,,,0.1,3.9,5,0,6,6,5,6,,250,180,18,50,,,"DO_NOT_AIM, DO_NOT_CONSERVE,","utility10, missile1, rare_bp, diable_weapon_bp",,Diable Avionics,,Saturation,Medium,Poor,,,,,,,,11.24
+Micro Missile Launcher,diableavionics_microbuiltin,1,,500,3000,,50,,5,6,3,25,1,4,HIGH_EXPLOSIVE,,,0.1,3.9,5,0,6,6,5,6,,250,180,18,50,,,"SYSTEM, DO_NOT_AIM, DO_NOT_CONSERVE, DO_NOT_AIM","utility10, missile1",,Diable Avionics,,Saturation,Medium,Poor,,,,,,,,
+Micro Missile Cluster,diableavionics_microcluster,3,,7500,3000,,50,,5,0,,90,2.5,10,HIGH_EXPLOSIVE,,,0.2,3.8,15,0.1,6,6,5,6,,250,150,18,50,,,"SYSTEM, DO_NOT_AIM, DO_NOT_CONSERVE, DO_NOT_AIM",,,,,,,,,,,,,,,
+Micro Missile Cluster,diableavionics_microcluster_left,3,,7500,3000,,50,,5,0,,90,2.5,10,HIGH_EXPLOSIVE,,,0.2,3.8,15,0.1,6,6,5,6,,250,150,18,50,,,"SYSTEM, DO_NOT_AIM, DO_NOT_CONSERVE, DO_NOT_AIM",,,,,,,,,,,,,,,
+Micro Missile Cluster,diableavionics_microcluster2,3,,7500,3000,,50,,5,0,,90,2.5,10,HIGH_EXPLOSIVE,,,0.2,3.8,15,0.1,6,6,5,6,,250,5,21,50,,,"SYSTEM, DO_NOT_AIM, DO_NOT_CONSERVE, DO_NOT_AIM",,,,,,,,,,,,,,,
+Micro Missile Array,diableavionics_microarray,3,,17500,3000,,50,,5,0,,960,8,32,HIGH_EXPLOSIVE,,,0.1,3.9,80,0.1,6,6,5,6,,250,150,18,50,,,"SYSTEM, DO_NOT_AIM, DO_NOT_CONSERVE, DO_NOT_AIM",,,,,,,,,,,,,,,
+Micro Missile Array,diableavionics_microarrayM,3,,17500,3000,,50,,5,0,,480,4,16,HIGH_EXPLOSIVE,,,0.1,3.9,40,0.1,6,6,5,6,,250,150,18,50,,,"SYSTEM, DO_NOT_AIM, DO_NOT_CONSERVE, DO_NOT_AIM",,,,,,,,,,,,,,,
```

Comment:
More missiles.

Reply:
All micromissile launchers range doubled (1500 -> 3000).
All micromissile launchers damage halved (100 -> 50).
All micromissile launchers missile lifetime increased to 18 (from 12). 
All micromissile launchers projectile speed reduced to 250 (from 300).
All micromissile launchers projectile hitpoints increased to 50 (from 25).
All micromissile launchers flux costs removed (from 50).

Micro Missile Launcher (small) max ammo increased to 25 (from 3.), ammo regeneration rate set to 1 per second (from 0.2 missiles per second), ammo reload size increased to 4 (from 3). this is effectively 4 ammo every 4 seconds. cooldown increased to 3.9 (from 2.9). burst size increased to 5 (from 1).

Micro Missile Cluster (on the Storm, Daze, Pocket Gust and the tier 1 and tier 2 fighter modules) max ammo increased to 90 (from 30), ammo regeneration rate set to 2.5 per second (from 0.5 missiles per second), ammo reload size increased to 10 (from 6). cooldown decreased to 3.8 (from 4). burst size increased to 15 (from 6).

Micro Missile Array (on the Maelstrom) max ammo increased to 960 (from 400), ammo regeneration rate set to 8 per second (from 1.5 missiles per second), ammo reload size decreased to 32 (from 40 but total ammo regen is still much faster). cooldown decreased to 3.9 (from 4). burst size increased to 80 (from 20).

Micro Missile Array M (on the Pandamonium and station modules) max ammo increased to 480 (from 200), ammo regeneration rate set to 4 per second (from 1.5 missiles per second), ammo reload size decreased to 16 (from 20 but total ammo regen is still much faster). cooldown decreased to 3.9 (from 4). burst size increased to 40 (from 10).